assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -90,6 +93,12 @@ function TemplateDetail() {
     }
   }, [fetchMyTemplateById, fetchExternalTemplateById, templateId, isLocal]);
 
+  const orderedTemplates = useMemo(() => {
```

Comment:
since I restructured templates to match stories for our restructured data in the reducer we need to do this to get the template array structure this view requires for paging through. I think eventually when we have an API we can clean this up.